### DISCUSSION TITLE
09:23:28 [Waring] (1534122677] app/proxyman/outbound: failed to process outbound traffic > proxy/vmess/ outbound: connection ends > proxy/vmess/ outbound: failed to read header > proxy/vmess/ encoding: failed to read response header > websocket: close 1000 (normal)

### DISCUSSION THREAD

**Discussion by praiman99** [v2fly/v2ray-core#1508] [General]
09:23:28 [Waring] (1534122677] app/proxyman/outbound:
failed to process outbound traffic > proxy/vmess/ outbound: connection ends > proxy/vmess/ outbound: failed to read header > proxy/vmess/ encoding: failed to read response header > websocket: close 1000 (normal)

**Comment by savaza** [comment]
Change client alterId to "0".

**Reply by praiman99** [reply]
Alright thanks sir for help

**Reply by iammsf** [reply]
> Change client alterId to "0".

Sooooooooooooooooooooooooooooooooo much appreciate
ive tried every thing i could but not worked. until i found this.
fixed, working well.

**Reply by Andreichenko** [reply]
> Change client alterId to "0".

It really helped in my connection situation.

**Reply by haodayizhia** [reply]
It's useful!!!!

**Reply by Long-Ni** [reply]
Yes!!! But why?my server config alterId is 64

**Reply by sprslm** [reply]
@savaza could you please help me how to change it sir ? I mean where should I go ? I have trouble with connecting with nekoray and it shows me this exact error

**Reply by MohammadGhaderi0** [reply]
> @savaza could you please help me how to change it sir ? I mean where should I go ? I have trouble with connecting with nekoray and it shows me this exact error

To change it in nekoray 
1. Click on your config
2. In the Edit window you can change alter id

**Reply by btlulu** [reply]
Same issue and it worked!

**Reply by itzenzy2** [reply]
not all heroes wear capes

**Comment by zhang-cuishan** [comment]
确实，修改alterid为0就可以了！客户端有两处可以修改：

1. ~\v2rayN-Core\config.json里面找
2. 在v2rayN客户端，双击服务器，弹窗找到额外id

**Comment by behniafb** [comment]
My alterId was already "0", & it didn't work again.

Solution:
I figured out that my Computer's clock is not synchronized; so I synchronized it, and it worked! ✅

**Reply by Darkrozee** [reply]
literal lifesaver right here <3

**Reply by 0b9** [reply]
Thank you so much, you're a real lifesaver, been trying to fix this for the past 8 hours, deserves much more attention

**Comment by ahmadidev** [comment]
Is it safe to set alterId to 0?

**Reply by 0b9** [reply]
Yes

**Comment by 372329332** [comment]
1. update system time

2. set Environment="V2RAY_VMESS_AEAD_FORCED=false"
vi  /etc/systemd/system/v2ray.service
[Unit]
Description=V2Ray Service
Documentation=https://www.v2fly.org/
After=network.target nss-lookup.target

[Service]
User=nobody
**Environment="V2RAY_VMESS_AEAD_FORCED=false"**
CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
AmbientCapabilities=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
NoNewPrivileges=true
ExecStart=/usr/bin/v2ray -config /etc/v2ray/config.json
Restart=on-failure
RestartPreventExitStatus=23

[Install]
WantedBy=multi-user.target

3. reload systemd service
systemctl daemon-reload

4. restart v2ary service
systemctl restart v2ray

hope useful for you

**Reply by hack3rcon** [reply]
Hello,
These solutions did not work for my system!

**Comment by Mr-persian** [comment]
2024/02/02 18:48:42 127.0.0.1:8072 accepted //github.com:443 [http -> proxy]
2024/02/02 18:48:42 127.0.0.1:8073 accepted //github.com:443 [http -> proxy]
2024/02/02 18:50:04 [Warning] [3180605360] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 188.114.97.3:12171: i/o timeout] > common/retry: all retry attempts failed
2024/02/02 18:50:04 [Warning] [3734704463] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 188.114.97.3:12171: i/o timeout] > common/retry: all retry attempts failed
2024/02/02 18:50:28 127.0.0.1:8094 accepted //a.nel.cloudflare.com:443 [http -> proxy]
2024/02/02 18:50:51 127.0.0.1:8098 accepted //github.com:443 [http -> proxy]
2024/02/02 18:51:00 127.0.0.1:8102 accepted //clients4.google.com:443 [http -> proxy]

same problem after set dns zone to proxied type .what should i do ???

**Comment by luogeshibu** [comment]
2024/11/28 15:33:43 [Warning] [3918294929] app/proxyman/outbound: failed to process outbound traffic > proxy/vmess/outbound: failed to find an available destination > common/retry: [transport/internet/websocket: failed to dial WebSocket > transport/internet/websocket: failed to dial to (wss://www.luogexiwen.top/ray): 502 Bad Gateway > websocket: bad handshake] > common/retry: all retry attempts failed
2024/11/28 15:33:45 [Warning] [2455995792] app/proxyman/outbound: failed to process outbound traffic > proxy/vmess/outbound: failed to find an available destination > common/retry: [transport/internet/websocket: failed to dial WebSocket > transport/internet/websocket: failed to dial to (wss://www.luogexiwen.top/ray): 502 Bad Gateway > websocket: bad handshake] > common/retry: all retry attempts failed
2024/11/28 15:34:03 127.0.0.1:53112 accepted //access-point.cloudmessaging.edge.microsoft.com:443 [http -> proxy]
2024/11/28 15:34:08 [Warning] [3729524286] app/proxyman/outbound: failed to process outbound traffic > proxy/vmess/outbound: failed to find an available destination > common/retry: [transport/internet/websocket: failed to dial WebSocket > transport/internet/websocket: failed to dial to (wss://www.luogexiwen.top/ray): 502 Bad Gateway > websocket: bad handshake] > common/retry: all retry attempts failed
2024/11/28 15:34:08 127.0.0.1:53118 accepted //access-point.cloudmessaging.edge.microsoft.com:443 [http -> proxy]
